src/pystack/_pystack/mem.h

```diff
@@ -227,11 +232,11 @@ class CorefileRemoteMemoryManager : public AbstractRemoteMemoryManager
     std::shared_ptr<CoreFileAnalyzer> d_analyzer;
     std::vector<VirtualMap> d_vmaps;
     std::vector<SimpleVirtualMap> d_shared_libs;
+    char* corefile_data;
+    size_t corefile_size;
```

Comment:
The other member data have a `d_` prefix on their names. We should match that naming convention here:
```suggestion
    char* d_corefile_data;
    size_t d_corefile_size;
```